### PR TITLE
Bug 1737374 Add new Suggest data prefs to clients_daily etc

### PR DESCRIPTION
**Pull request by jklukas**
See https://bugzilla.mozilla.org/show_bug.cgi?id=1737374#c3

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] If the PR comes from a fork, trigger integration CI tests by running the [Push to upstream workflow](https://github.com/mozilla/bigquery-etl/actions/workflows/push-to-upstream.yml) and provide the `<username>:<branch>` of the fork as parameter. The parameter will also show up
in the logs of the `manual-trigger-required-for-fork` CI task together with more detailed instructions.
- [ ] If adding a new field to a query, ensure that the schema and dependent downstream schemas have been updated

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql [hunk 1]
```diff
@@ -47,10 +47,14 @@ WITH combined_urlbar_picked AS (
...
     ) AS show_search_suggestions_first,
     SAFE_CAST(user_pref_browser_urlbar_suggest_quicksuggest AS BOOL) AS suggest_quicksuggest,
+    SAFE_CAST(
+      user_pref_browser_urlbar_suggest_quicksuggest_nonsponsored AS BOOL
+    ) AS suggest_quicksuggest_nonsponsored,
     SAFE_CAST(
       user_pref_browser_urlbar_suggest_quicksuggest_sponsored AS BOOL
     ) AS suggest_quicksuggest_sponsored,
     user_pref_browser_urlbar_quicksuggest_onboarding_dialog_choice AS quicksuggest_onboarding_dialog_choice,
+    user_pref_browser_urlbar_quicksuggest_data_collection_enabled AS quicksuggest_data_collection_enabled,
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql:57]: @rebecca-burwei Do you want to do anything special with the naming in urlbar_clients_daily here?

**Comment by jklukas** [comment]
Dry run failure is expected. Once approved, I will push the schema changes and verify that dry run succeeds before merging.

**Review by rebecca-burwei — APPROVED** [review]
looks good to me. my only request is to have some documentation about what these new preferences mean and documentation for old preferences whose meaning has changed. I know Drew is also working on this.

File: sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql (hunk 1 above)
Comment (rebecca-burwei) [reply sql/moz-fx-data-shared-prod/telemetry_derived/urlbar_clients_daily_v1/query.sql:57]: no, personally, I prefer consistency in names across datasets over choosing a more meaningful, but different name in each derived table. better for automation in my downstream code.

File: sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql [hunk 2]
```diff
@@ -484,12 +484,18 @@ clients_summary AS (
...
+        )[SAFE_OFFSET(0)] AS user_pref_browser_urlbar_suggest_quicksuggest_nonsponsored,
         ARRAY_AGG(
           IF(key = 'browser.urlbar.suggest.quicksuggest.sponsored', value, NULL) IGNORE NULLS
         )[SAFE_OFFSET(0)] AS user_pref_browser_urlbar_suggest_quicksuggest_sponsored,
         ARRAY_AGG(
           IF(key = 'browser.urlbar.quicksuggest.onboardingDialogChoice', value, NULL) IGNORE NULLS
         )[SAFE_OFFSET(0)] AS user_pref_browser_urlbar_quicksuggest_onboarding_dialog_choice,
+        ARRAY_AGG(
+          IF(key = 'browser.urlbar.quicksuggest.dataCollection.enabled', value, NULL) IGNORE NULLS
+        )[SAFE_OFFSET(0)] AS user_pref_browser_urlbar_quicksuggest_data_collection_enabled,
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql:498]:
```suggestion
        -- New pref for Firefox Suggest introduced in bug 1737374
        ARRAY_AGG(
          IF(key = 'browser.urlbar.quicksuggest.dataCollection.enabled', value, NULL) IGNORE NULLS
        )[SAFE_OFFSET(0)] AS user_pref_browser_urlbar_quicksuggest_data_collection_enabled,
```

File: sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql [hunk 3]
```diff
@@ -484,12 +484,18 @@ clients_summary AS (
         ARRAY_AGG(IF(key = 'browser.urlbar.suggest.quicksuggest', value, NULL) IGNORE NULLS)[
           SAFE_OFFSET(0)
         ] AS user_pref_browser_urlbar_suggest_quicksuggest,
+        ARRAY_AGG(
+          IF(key = 'browser.urlbar.suggest.quicksuggest.nonsponsored', value, NULL) IGNORE NULLS
+        )[SAFE_OFFSET(0)] AS user_pref_browser_urlbar_suggest_quicksuggest_nonsponsored,
```
Comment (jklukas) [inline sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql:489]:
```suggestion
        -- Rename of browser.urlbar.suggest.quicksuggest.nonsponsored; see bug 1737374
        ARRAY_AGG(
          IF(key = 'browser.urlbar.suggest.quicksuggest.nonsponsored', value, NULL) IGNORE NULLS
        )[SAFE_OFFSET(0)] AS user_pref_browser_urlbar_suggest_quicksuggest_nonsponsored,
```

**Pull request merged**